docs/application/dotnet/guides/nui/text.md

```diff
@@ -0,0 +1,575 @@
+
+    |  Alignment  | Left-to-right script example    |      Right-to-left script example |
+    |-------------|---------------------------------|-----------------------------------|
+    |  `Begin` |   ![Latin script aligned to beginning](media/LatinBegin.png) |  ![Arabic script aligned to beginning](media/ArabicBegin.png) |
+    |`Center` |  ![Latin script aligned to center](media/LatinCenter.png) | ![Arabic script aligned to center](media/ArabicCenter.png) |
+    | `End`   |  ![Latin script aligned to end](media/LatinEnd.png) |   ![Arabic script aligned to end](media/ArabicEnd.png) |
+
+
+
+<a name="decorations"></a>
+### Use Decorations
+
+For text decorations, the [Tizen.NUI.BaseComponents.TextLabel](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.TextLabel.html) class provides several properties. All the properties are editable and none of them are animatable:
+
+
+| Property    |    Type |      Description |
+|-------------|---------|------------------|
+| `Text` |  String |     Specifies the text to display in UTF-8 format.|
+| `FontFamily` | String |   Specifies the requested font family. |
+|  `FontStyle` |  Map |      Specifies the requested font style. |
+|  `MultiLine` |  Boolean  |     Specifies whether or not to use the multi-line layout option.  |
+|  `HorizontalAlignment` | String | Specifies the horizontal line alignment. |
+|  `VerticalAlignment` | String | Specifies the vertical line alignment. |
+| `TextColor`  |   Color |  Specifies the color of the text. |
+|  `ShadowOffset` | Vector2 |  Specifies the shadow offset of the text. |
+|  `ShadowColor` | Vector4  |  Specifies the shadow color of the text. |
+| `UnderlineEnabled` | Bool |  Specifies whether or not to use underline.  |
+| `UnderlineColor` | Vector4 | Specifies the underline color of the text. |
+| `UnderlineHeight` |  Float |  Specifies the height of the underline. |
+|  `EnableMarkup` |  Boolean |  Specifies whether to enable or disable the markup string to process text within the markup tags using DALi application.<br>**Note**: By default, the markup string is disabled. |
+| `EnableAutoScroll` |  Boolean | Specifies whether to enable or disable the auto scrolling. |
+| `AutoScrollSpeed` | Integer  | Specifies the scrolling speed in pixels per second.  |
+| `AutoScrollLoopCount` | Integer |    Specifies the number of complete loops to scroll, when scrolling is enabled. |
+| `AutoScrollGap` | Float |   Specifies the gap before scrolling wraps. |
+| `LineSpacing` |   Float | Specifies the default spacing between lines in points. |
+|  `Underline` |   Map |  Specifies the default underline parameters. |
+| `Shadow` | Map |      Specifies the default shadow parameters. |
+|  `Emboss` |  String  |         Specifies the default emboss parameters. |
+| `Outline` |  Map  |         Specifies the default outline parameters. |

```

Comment:
It looks like [this comment](https://github.com/Samsung/tizen-docs/pull/1009#discussion_r394804628) is not applied.

>Please use PropertyMap instead of Map as like the table case for TextField and TextEditor.

Please change the Type of `FontStyle`, `Underline`, `Shadow`, and `Outline` to PropertyMap.

Reply:
Reminder of https://github.com/Samsung/tizen-docs/pull/1009/files#r396215282